kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# This is an example of how to define a Domain resource.  Please read through the comments which explain
+# what updates are needed.
+#
+apiVersion: "weblogic.oracle/v2"
+kind: Domain
+metadata:
+  # Update this with the `domainUID` of your domain:
+  name: domain1
+  # Update this with the namespace your domain will run in:
+  namespace: weblogic
+  labels:
+    weblogic.resourceVersion: domain-v2
+    # Update this with the `domainUID` of your domain:
+    weblogic.domainUID: domain1
+
+spec:
+  # This parameter provides the location of the WebLogic Domain Home (from the container's point of view).
+  # Note that this might be in the image itself or in a mounted volume or network storage.
+  domainHome: /u01/oracle/user_projects/domains/domain1
+
+  # If the domain home is inside the Docker image, set this to `true`, otherwise set `false`:
+  domainHomeInImage: true
+
+  # Update this with the name of the Docker image that will be used to run your domain:
+  image: "my-domain1-image:1.0"
+
+  # imagePullPolicy defaults to "Always" if image version is :latest
+  imagePullPolicy: "IfNotPresent"
+
+  # If credentials are needed to pull the image, uncomment this section and identify which
+  # Secret contains the credentials for pulling an image:
+  #imagePullSecrets:
+  #- name:
+
+  # Identify which Secret contains the WebLogic Admin credentials (note that there is an example of
+  # how to create that Secret at the end of this file)
+  webLogicCredentialsSecret:
+    # Update this with the name of the secret containing your WebLogic server boot credentials:
+    name: domain1-weblogic-credentials
+
+  # If you want to include the server out file into the pod's stdout, set this to `true`:
+  includeServerOutInPodLog: true
+
+  # If you want to use a mounted volume as the log home, i.e. to persist logs outside the container, then
+  # uncomment this and set it to `true`:
+  # logHomeEnabled: false
+  # The in-pod name of the directory to store the domain, node manager, server logs, and server .out
+  # files in.
+  # If not specified or empty, domain log file, server logs, server out, and node manager log files
+  # will be stored in the default logHome location of /shared/logs/<domainUID>/.
```

Comment:
Shall we add a comment saying something like "Update this with the desired location for the logs."